docs/zz_generated.seed.yaml

```diff
@@ -149,6 +149,10 @@ spec:
           # See https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#block-storage
           # This setting defaults to false.
           trust_device_path: false
+          # Optional: Gets mapped to the "use-octavia" setting in the cloud config.
```

Comment:
Probably you forgot `hack/update-docs.sh` this time ;-)

Reply:
Yes, I am on it. Absolutely a pain! ;-)
`make update` would do the trick.

Reply:
There is room for improvement :-) don't hesitate to open issues or create PRs.